### PR TITLE
Restyle blog to adhere to figma design

### PR DESCRIPTION
**Pull request by jackson-harris-iii**
Add 3 demo blog posts

### CODE REVIEW

File: src/BlogPostPage.jsx [hunk 1]
```diff
@@ -87,21 +298,173 @@ if (blog) {
...
+                props={{
+                  item,
+                  onClick: () => State.update({ showReply: !state.showReply }),
+                }}
+              />
+            </PromptSignUpWrapper>
+            <Widget
+              src="${REPL_ACCOUNT}/widget/CopyUrlButton"
+              props={{
+                url: postUrl,
```
Comment (charleslavon) [inline src/BlogPostPage.jsx:441]: I'm seeing a null in the UI when I try to copy or share. Looks like `postUrl` is never set

**Comment by charleslavon** [comment]
@jackson-harris-iii nice job parsing the markdown and formatting the UI. It's looking good

File: src/BlogPostPage.jsx (hunk 1 above)
Comment (jackson-harris-iii) [reply src/BlogPostPage.jsx:441]: ah, that's part of what's in flight. buttoned that up and working on getting the other actions going along with the sign in feature.

File: src/BlogPostPage.jsx [hunk 2]
```diff
@@ -87,20 +298,183 @@ if (blog) {
...
+            iconLeft: "ph ph-arrow-left",
+            variant: "secondary",
+            size: "small",
+          }}
+          style={{ marginBottom: "1rem" }}
+        />
+        {/* RENDER BLOG HEADER IMAGE */}
+        <PostImage
+          imageUrl={
+            markdownObj[0].imageUrl || "https://pages.near.org/wp-content/uploads/2023/06/generic-green-blog.png"
```
Comment (charleslavon) [inline src/BlogPostPage.jsx:344]: Nice default. This is fine for now, but please track moving this to ipfs so we do not increase our dependency on pages.near.org.

File: src/BlogPostPage.jsx [hunk 3]
```diff
@@ -87,20 +298,183 @@ if (blog) {
...
+          <TitleSection>
+            {/* RENDER BLOG HEADER - TITLE   */}
+
+            <h1>{getFirstHeading(markdownObj)?.text || "Untitled"}</h1>
+
+            {/* RENDER BLOG HEADER - DATE */}
+
+            <p>
+              <Widget
+                src="${REPL_ACCOUNT}/widget/TimeAgo"
```
Comment (charleslavon) [inline src/BlogPostPage.jsx:358]: @jackson-harris-iii can we easily show the non abbreviated date? like March 1, 2024? 
Screen Shot 2024-03-12 at 9 31 38 AM

**Review by charleslavon — APPROVED** [review]
A couple minor comments to address but this looks good 👍🏾

File: src/BlogPostPage.jsx (hunk 2 above)
Comment (jackson-harris-iii) [reply src/BlogPostPage.jsx]: moved to ipfs

**Pull request merged**